market_maker_keeper/uniswapv2_market_maker_keeper.py

```diff
@@ -190,12 +185,34 @@ def main(self):
             lifecycle.on_shutdown(self.shutdown)
 
     def startup(self):
+        self.plunge()
         self.uniswap.approve(self.token_a)
         self.uniswap.approve(self.token_b)
 
     def shutdown(self):
         self.logger.info(f"Shutdown notification received, removing all available liquidity")
-        self.remove_liquidity()
+        self.remove_liquidity(True)
+
+    def plunge(self):
```

Comment:
Is there a reason this class shouldn't inherit from `DEXKeeperAPI`, which offers a `plunge` method?

Reply:
Yeah I added it to both as `DEXKeeperAPI` is largely built around an order book model. I figured it wasn't really a great fit for Uniswap as an AMM, and we could add a different AMM based abstract class following future AMM integrations.